apps/cli/src/commands/create.ts

```diff
@@ -34,23 +35,23 @@ export default class CreateCommand extends Command {
                 "typescript",
             ],
         }),
-        branch: Flags.string({
-            description: `sunodo/sunodo-templates repository branch name to use`,
-            default: DEFAULT_TEMPLATES_BRANCH,
-        }),
+        repository: Flags.string({
```

Comment:
There's no reason to stop supporting the `--branch` flag.

There's an argument for hiding it here https://github.com/sunodo/sunodo/issues/186#issuecomment-1962992079